docs/Microservices/overview.md

```diff
@@ -3,47 +3,66 @@ Figure 1 illustrates the high-level architecture of Open Active Management Techn
 4. **UI Toolkit** - A toolkit that includes prebuilt React components and a reference implementation web console. The React-based snippets simplify the task of adding complex manageability-related UI controls, such as the KVM, to a console. 
-5. **Sample Web UI** - A web based UI that demonstrates how to use the UI-Toolkit. It also provides a way to interact with the microservices and to help provide context as to how each microservice is used.
+5. **Sample Web UI** - A web based UI that demonstrates how to use the UI Toolkit. It also provides a way to interact with the microservices and to help provide context as to how each microservice is used.
 
 
-## Out-of-band (OOB) Management 
+## Out-of-band Management (OOBM)
 
-Open AMT Cloud Toolkit uses remote management technology, also known as out-of-band (OOB) management, to allow administrators to perform actions on network assets or devices using a secure alternative to LAN-based communication protocols. Actions include reboot, power up, power down, system updates, and more. As long as the network device or asset is connected to power, Open AMT Cloud Toolkit software can perform remote management, including powering up as a system that is currently powered down.   
+Open AMT Cloud Toolkit uses remote management technology, also known as [out-of-band management (OOBM)](../Glossary.md#o), to allow administrators to perform actions on network assets or devices using a secure alternative to LAN-based communication protocols. Actions include reboot, power up, power down, system updates, and more. As long as the network device or asset is connected to power, Open AMT Cloud Toolkit software can perform remote management, including powering up as a system that is currently powered down.  
+
+Remote management can offer potential cost-savings by decreasing the need for in-person technician visits to remote IT sites and reducing downtime.
+
+## What's the difference between in-band and out-of-band management?
+
+Remote monitoring and management software solutions often require the managed devices to be in the powered on state. The IT administrator connects to and updates the managed device while it is in the powered on state.
+
+With out-of-band management, the administrator can connect to the device when it has been powered down or it is unresponsive. 
 
 ### CIRA Configuration
 
 CIRA enables OOB connections between Intel® AMT platforms and administrative development systems running Open AMT on the same network. 
 
 The [following steps](https://01.org/open-active-management-technology-cloud-toolkit/overview/management-presence-server) occur via a CIRA channel:
 
-1. A remote Intel vPro® platform featuring Intel® AMT is provisioned with CIRA enabled. The remote platform is referred to as the managed device. 
+1. A remote Intel vPro® platform featuring Intel® AMT is is activated and a CIRA configuration is applied. The remote platform is referred to as the managed device. 
```

Comment:
@kimberlyxdavis there are 2 'is' after Intel(r) AMT

Reply:
Modified.